draft-ietf-rats-corim.md

```diff
@@ -777,6 +777,9 @@ context (triple) in which the environment is used.
 An environment is named after a class, instance or group identifier (or a
 combination thereof).
 
+An environment MUST be globally unique.
```

Comment:
For example, an environment with just `class-id` may denote a specific organization's SKU (product name and specifications), the `instance` field is unique within the `class-id` namespace.